kernel/dw1000.c

```diff
@@ -1274,8 +1274,6 @@ static int dw1000_ptp_adjfine(struct ptp_clock_info *ptp, long delta)
 	hires_counter_setmult(tc, mult);
 	mutex_unlock(&dw->ptp.mutex);
 
-	dev_dbg(dw->dev, "adjust frequency %+ld sppm: multiplier adj %llu\n",
-		delta, (unsigned long long) mult);
```

Comment:
Any reason why this is removed as part of this commit?

Reply:
none